covalent/_results_manager/results_manager.py

```diff
@@ -69,12 +64,12 @@ def get_result(dispatch_id: str, wait: bool = False) -> Result:
         )
         result_object = pickle.loads(codecs.decode(result["result"].encode(), "base64"))
 
-    except MissingLatticeRecordError as e:
-        app_log.warning(
-            f"Dispatch ID {dispatch_id} was not found in the database. Either the Dispatch ID is incorrect or wait a couple of seconds before trying again."
+    except MissingLatticeRecordError:
+        app_log.debug(
+            f"Dispatch ID {dispatch_id} was not found in the database. Incorrect dispatch id."
         )
 
-        time.sleep(0.01)
+        time.sleep(0.1)
         return get_result(dispatch_id, wait)
```

Comment:
Related to above... if the `404` is returned correctly, then I think we shouldn't be doing `get_result` again since it will keep `404`ing

Reply:
Hi @scottwn , this is tracked in the followup issue #926. I'm rolling the fix for that into this branch.